src/locales/en.json

```diff
@@ -625,5 +625,14 @@
     "see-image": "See all images",
     "see-audio": "See all audio"
   },
-  "skip-to-content": "Skip to content"
+  "skip-to-content": "Skip to content",
+  "pref-page": {
```

Comment:
🤔 How will we preview "switchable" features to our audience?
🎨 This might need to be an actual page in our designs.

Reply:
We could add a note that says that the disabled checkboxes are not switchable for the current environment, or something like that?

Reply:
Updated the text. The UI should be much clearer now.